rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ModelObjectBoilerPlate.xtend

```diff
@@ -75,7 +73,7 @@ class ModelObjectBoilerPlate {
 	private def StringConcatenationClient contributeHashCode(RAttribute attr, JavaScope scope) {
 		val id = scope.getIdentifierOrThrow(attr)
 		'''
-			«IF attr.RType instanceof REnumType»
+			«IF attr.RMetaAnnotatedType.RType instanceof REnumType»
```

Comment:
Hm, actually this should also check that the attribute does not have any metadata.

FYI, this is one example where the R-layer unjustly flows into the Java layer. The right check would be `IF attrJavaType instanceof RJavaEnum`.

Reply:
done